<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OSHaxiswca</class>
 <widget class="QWidget" name="OSHaxiswca">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>40</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QProgressBar" name="AxisValue">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>20</y>
     <width>491</width>
     <height>8</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>6</pointsize>
    </font>
   </property>
   <property name="maximum">
    <number>4095</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="textVisible">
    <bool>true</bool>
   </property>
   <property name="format">
    <string>%v</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_MinCalib">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="inputMask">
    <string>9999</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_MaxCalib">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>0</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="inputMask">
    <string>9999</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>20</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>MinCalib</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>20</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>MaxCalib</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_AutoCalib">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>0</y>
     <width>111</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>AutoCalibration</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
